src/main/resources/config/liquibase/changelog/20200625120000_increase_validation_feedback.xml

```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0" encoding="utf-8"?>
+<databaseChangeLog
+    xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+    xmlns:ext="http://www.liquibase.org/xml/ns/dbchangelog-ext"
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.6.xsd
+                        http://www.liquibase.org/xml/ns/dbchangelog-ext http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-ext.xsd">
+
+    <changeSet author="bmatern" id="increaseValidationFieldSize">
+    
+        <modifyDataType
+        columnName="validation_feedback"
+        newDataType="varchar(3000)"
```

Comment:
@bmatern did you check if that's working (with mariadb/mysql)? Sounds like we need a TEXT type or something...

Reply:
I didn't check with a different database type, you mean a TEXT type instead of varchar?

Reply:
I see what you mean, sometimes TEXT is used, like in the project description, but varchar(xx) is also used in a few places. I think it makes sense to switch it to text

Reply:
Seems to be working...

Reply:
Let's go for TEXT - this seems to be the safer and more compatible with other RDBMS.